tests/servers/tests.py

```diff
@@ -27,6 +29,34 @@
 }
 
 
+class LiveServerThreadTest(TestCase):
+
+    def run_live_server_thread(self, connections_override=None):
+        thread = LiveServerTestCase._create_server_thread(connections_override)
+
+        thread.daemon = True
+        thread.start()
+        thread.is_ready.wait()
+        thread.terminate()
+        thread.join()
+
+    @unittest.skipIf(connection.vendor == 'sqlite', "sqlite does not permit closing connections")
```

Comment:
I believe the test shouldn't be skipped on SQLite, it's failing locally because you're closing overridden connections which are used when using in-memory SQLite databases. You should be able to remove this `skipIf` after adjusting the `finally` block to only close non-overriden connections.